superset/models/slice.py

```diff
@@ -53,9 +53,9 @@
 logger = logging.getLogger(__name__)
 
 
-class Slice(
+class Slice(  # pylint: disable=too-many-instance-attributes,too-many-public-methods
```

Comment:
I'm not sure why `pylint` was complaining on this file as it _was_ unchanged.

Reply:
Yes, I also faced this issue.

Reply:
Same here. :-P